Sources/SwiftPolyglotCore/SwiftPolyglotError.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+enum SwiftPolyglotError: Error {
+    case missingTranslations
+    case noLanguageCodes
+}
+
+extension SwiftPolyglotError: LocalizedError {
+    public var errorDescription: String? {
+        switch self {
+        case .missingTranslations:
+            "Error: One or more translations are missing."
+        case .noLanguageCodes:
+            "Usage: script.swift <language codes> [--errorOnMissing]"
```

Comment:
nit: should this reference the actual script to run, rather than just `script.swift`?

Reply:
Good catch!! Fixed!!